### PR TITLE
fix(FEC-12189): Stream doesn't start if open a program with a bookmark

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

Solves FEC-12189

 pr: 
https://github.com/kaltura/playkit-js-dash/pull/194

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/player-config.js [hunk 1]
```diff
@@ -59,8 +59,7 @@ const DefaultConfig = {
     ]
   },
   streaming: {
-    forceBreakStall: false,
-    lowLatencyMode: true
+    forceBreakStall: false
```
Comment (SivanA-Kaltura) [inline src/player-config.js:62]: why not lowLatencyMode: false ?

Comment (JonathanTGold) [reply src/player-config.js:62]: https://shaka-player-demo.appspot.com/docs/api/shaka.extern.html#.StreamingConfiguration
it is false by default

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: src/player-config.js (hunk 1 above)
Comment (giladna) [reply src/player-config.js:62]: @JonathanTGold 
please add 
lowLatencyMode: false 

we must merge it  ASAP

Comment (JonathanTGold) [reply src/player-config.js:62]: Right ,  and it's false by default in shaka, everyone keeps its default.

File: docs/configuration.md [hunk 2]
```diff
@@ -1066,7 +1066,7 @@ var config = {
 > ```js
 > {
 >   forceBreakStall: false,
->   lowLatencyMode: true // false in smart TV
+>   //lowLatencyMode: (optional) true for hls, false in dash and smart TV
```
Comment (SivanA-Kaltura) [inline docs/configuration.md:1069]:
```suggestion
>   lowLatencyMode: (optional) 
     default true for hls, 
     default false for dash and smart TV
```

**Review by SivanA-Kaltura — DISMISSED** [review]

**Review by giladna — APPROVED** [review]

**Pull request merged**